pywwt/annotation.py

```diff
@@ -52,8 +58,17 @@ def _on_trait_change(self, changed):
 
 
 class Circle(Annotation):
+    """
+    Used to render a circle on screen (via pywwt.BaseWWTWidget.add_circle()) 
+    and edit its properties after intialization. Inherits from Annotation
+    class.
```

Comment:
You don't need to say 'Inherits from ...'

Also for ``pywwt.BaseWWTWidget.add_circle()``, write:

```
:meth:`pywwt.BaseWWTWidget.add_circle`
```

instead and that will auto-link to the API docs

Reply:
Same comment for the other shapes